packages/@aws-cdk/aws-codecommit/lib/repository.ts

```diff
@@ -306,13 +321,20 @@ export class Repository extends RepositoryBase {
       public repositoryArn = Repository.arnForLocalRepository(repositoryName, scope);
       public readonly repositoryCloneUrlHttp = Repository.makeCloneUrl(stack, repositoryName, 'https');
       public readonly repositoryCloneUrlSsh = Repository.makeCloneUrl(stack, repositoryName, 'ssh');
+      public readonly repositoryCloneUrlGrc = Repository.makeCloneUrl(stack, repositoryName, 'grc');
     }
 
     return new Import(scope, id);
   }
 
-  private static makeCloneUrl(stack: Stack, repositoryName: string, protocol: 'https' | 'ssh', region?: string) {
-    return `${protocol}://git-codecommit.${region || stack.region}.${stack.urlSuffix}/v1/repos/${repositoryName}`;
+  private static makeCloneUrl(stack: Stack, repositoryName: string, protocol: 'https' | 'ssh' | 'grc', region?: string) {
```

Comment:
While you're doing changes in the area - can you actually change this to be a `function` private to this file instead?